android/src/ti/crashlytics/TitaniumCrashlyticsModule.java

```diff
@@ -63,6 +63,31 @@ public void throwException()
 		}
 	}
 
+	@Kroll.method
+	public void throwCustomException(String message, String myline, String sourceNameAndroid, String lineSource, String javascriptStack)
+	{
+		Log.d(message);
```

Comment:
Does not compile like this (`no suitable method found for d(String)`), the first parameter has the be `LCAT` like in my comment before. Please compile it before pushing